Apps/WebAPI/sdLitica.WebAPI/Services/IInfluxDB.cs

```diff
@@ -0,0 +1,20 @@
+using System;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using Vibrant.InfluxDB.Client;
+using Vibrant.InfluxDB.Client.Rows;
+
+namespace sdLitica.WebAPI.Services
+{
+    public interface IInfluxDB
```

Comment:
Please, move this interface to ``sdLitica.Services`` project. Also, please, do not use technology in interfaces names. Consider rename this to something more abstract as: ``ITimeSeriesService``.